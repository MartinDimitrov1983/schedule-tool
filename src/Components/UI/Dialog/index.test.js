import React from 'react';
import { render } from '@testing-library/react';

import CustomDialog from './index';

test('Snapshot CustomDialog', () => {
  const { asFragment } = render(
    <CustomDialog isOpen={false} setIsOpen={() => {}} />,
  );
  expect(asFragment()).toMatchSnapshot();
});
