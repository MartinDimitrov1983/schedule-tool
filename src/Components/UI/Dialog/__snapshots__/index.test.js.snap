// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot CustomDialog 1`] = `<DocumentFragment />`;
